core/tutorials/Unloading/Plugin/Plugin.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Interface\Interface.csproj">
+      <Private>false</Private>
```

Comment:
Add a comment describing that this is the trick which makes the Interface.dll not present in the plugins directory. The name of the MSBuild property is misleading (to say the least).

Reply:
Added, thanks for the suggestion.